executor/point_get.go

```diff
@@ -322,19 +321,14 @@ func (e *PointGetExecutor) lockKeyIfNeeded(ctx context.Context, key []byte) erro
 	if e.lock {
 		seVars := e.ctx.GetSessionVars()
 		lockCtx := newLockCtx(seVars, e.lockWaitTime)
-		lockCtx.ReturnValues = true
-		lockCtx.Values = map[string]tikvstore.ReturnedValue{}
+		lockCtx.InitReturnValues(1)
```

Comment:
could we use 0 instead?

Reply:
Actually, I'm not very sure either. /cc @sticnarf could help to take a look?

Reply:
`1` looks reasonable to me.